aws/data_source_aws_rds_orderable_db_instance.go

```diff
@@ -31,15 +31,23 @@ func dataSourceAwsRdsOrderableDbInstance() *schema.Resource {
 				Computed: true,
 			},
 
+			"default_only": {
+				Type:          schema.TypeBool,
+				Optional:      true,
+				Default:       false,
```

Comment:
Extraneous default since it matches the zero-value for the type:

```suggestion
```

Reply:
On further thought, I don't like this argument and would rather if no version is given to enable `input.DefaultOnly`.